examples/3d/camera_sub_view.rs

```diff
@@ -0,0 +1,274 @@
+                },
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_systems(Startup, setup)
+        .add_systems(Update, move_camera_view)
+        .run();
+}
+
+#[derive(Debug, Component)]
+struct MovingCameraMarker;
+
+/// Set up a simple 3D scene
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    let transform = Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y);
+
+    // Plane
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Plane3d::default().mesh().size(5.0, 5.0)),
+        material: materials.add(Color::srgb(0.3, 0.5, 0.3)),
+        ..default()
+    });
+
+    // Cube
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Cuboid::default()),
+        material: materials.add(Color::srgb(0.8, 0.7, 0.6)),
+        transform: Transform::from_xyz(0.0, 0.5, 0.0),
+        ..default()
+    });
+
+    // Light
+    commands.spawn(PointLightBundle {
+        point_light: PointLight {
+            shadows_enabled: true,
+            ..default()
+        },
+        transform: Transform::from_xyz(4.0, 8.0, 4.0),
+        ..default()
+    });
+
+    // Main perspective Camera
+    commands.spawn(Camera3dBundle {
+        camera: Camera {
+            viewport: Option::from(Viewport {
+                physical_size: UVec2::new(LARGE_SIZE, LARGE_SIZE),
+                physical_position: UVec2::new(PADDING, PADDING * 2 + SMALL_SIZE),
+                ..default()
+            }),
+            ..default()
+        },
+        transform,
+        ..default()
+    });
+
+    // Perspective camera left half
+    commands.spawn(Camera3dBundle {
+        camera: Camera {
+            viewport: Option::from(Viewport {
+                physical_size: uvec2(SMALL_SIZE, SMALL_SIZE),
+                physical_position: uvec2(PADDING, PADDING),
+                ..default()
+            }),
+            sub_camera_view: Some(SubCameraView {
+                // Set the sub view camera to the left half of the full image
+                full_size: uvec2(500, 500),
+                offset: ivec2(0, 0),
+                size: uvec2(250, 500),
+            }),
+            order: 1,
+            ..default()
+        },
+        transform,
+        ..default()
+    });
+
+    // Perspective camera moving
+    commands.spawn((
+        Camera3dBundle {
+            camera: Camera {
+                viewport: Option::from(Viewport {
+                    physical_size: uvec2(SMALL_SIZE, SMALL_SIZE),
+                    physical_position: uvec2(PADDING * 2 + SMALL_SIZE, PADDING),
+                    ..default()
+                }),
+                sub_camera_view: Some(SubCameraView {
+                    // Set the sub view camera to a fifth of the full view and
+                    // move it in another system
+                    full_size: uvec2(500, 500),
```

Comment:
Will let @alice-i-cecile make a determination, but we tend not to use these function constructors anywhere in engine code (although I personally find them quite nice!)

Reply:
Yeah, let's keep it consistent and use the non-function forms.